Core/GDCore/Extensions/Builtin/MouseExtension.cpp

```diff
@@ -38,6 +38,32 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsMouseExtension(
       .AddCodeOnlyParameter("conditionInverted", "")
       .MarkAsSimple();
 
+  extension
+      .AddCondition("ScrollUp",
```

Comment:
Let's be more precise: "IsMouseWheelScrollingUp" and same for the other condition. In the future there might be other way to scroll up/down